saleor/graphql/translations/resolvers.py

```diff
@@ -40,7 +44,7 @@ def resolve_translation(instance, info, language_code):
 
     loader = TYPE_TO_TRANSLATION_LOADER_MAP.get(type(instance))
     if loader:
-        return loader(info.context).load((instance.pk, language_code))
+        return loader(info.context).load((instance.id, language_code))
```

Comment:
Checkout is the only object with a custom PK field name and it is not translatable.